src/activities/drawletters/drawletters.qrc~

```diff
@@ -0,0 +1,48 @@
+<RCC>
```

Comment:
You commited src/activities/drawletters/drawletters.qrc~ and src/activities/drawletters/resource/README~ which are temporary files (the ~ at the end). Both should be deleted (but not README).
drawletters.qrc should also not be commited (it is not) because it is generated, don't try to commit it ;)